libr/anal/esil.c

```diff
@@ -2914,6 +2916,8 @@ R_API int r_anal_esil_parse(RAnalEsil *esil, const char *str) {
 	esil->skip = 0;
 	esil->parse_goto = -1;
 	esil->parse_stop = 0;
+// memleak or failing aetr test. wat du
+//	r_anal_esil_stack_free (esil);
```

Comment:
We can ignore `aetr` after all, it is kind of incomplete and of low importance: https://github.com/radare/radare2/issues/9865

Reply:
poor reil